test/gtest/uct/ib/test_ib.cc

```diff
@@ -9,8 +9,13 @@
 test_uct_ib::test_uct_ib() : m_e1(NULL), m_e2(NULL) { }
 
 void test_uct_ib::create_connected_entities() {
-    m_e1 = uct_test::create_entity(0);
-    m_e2 = uct_test::create_entity(0);
+    if (m_e1 == NULL) {
```

Comment:
I think it was in another PR - no need to call this function if m_e1,m_e2 are already initialized